lib/challenge.js

```diff
@@ -254,6 +293,12 @@ function prompt11() {
 
 function prompt12() {
   // YOUR CODE HERE
```

Comment:
```
for (let i = 1; i <= 10; i++) {
    for (let j = 11; j <= 20; j++) {
     console.log(`i: ${i} / j: ${j}`)
   }
}
```

It looks like `i` was out of scope. To nest a loop, keep the contents of the second for-loop with the `{ brackets }` of the first for-loop